README.md

```diff
@@ -226,3 +226,17 @@ import { like } from '../my-rsql-operators/like';
 
 const op = like('John Travolta'); // '=like="John Travolta"'
 ```
+
+If the value can contain special characters, `escapeValue` function can be used to escape them:
```

Comment:
I would say, that it's best practice by default. I mean using `escapevalue` function. Could you pls update example above

Reply:
Updated.